listenbrainz/db/dump.py

```diff
@@ -631,12 +632,20 @@ def copy_table(cursor, location, columns, table_name):
                      that should be dumped
             table_name: the name of the table to be copied
     """
+    fields = []
+    for column in columns:
```

Comment:
perhaps a small doc here too explaining the difference between a composable and an Identifier (perhaps just one doc for these two items?)

Reply:
Added